Leanplum/Manager/LPRequestQueue.m

```diff
@@ -11,24 +11,40 @@
 #import "LPAPIConfig.h"
 #import "LPConstants.h"
 #import "LPRequestManager.h"
+#import "LPRequestCallbackManager.h"
+
+@interface LPRequestQueue()
+
+@property (nonatomic, strong) LPRequestCallbackManager *callbackManager;
+
+@end
 
 @implementation LPRequestQueue
 
 + (instancetype)sharedInstance {
     static LPRequestQueue *sharedManager = nil;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        sharedManager = [[self alloc] init];
+        sharedManager = [[self alloc] initWithCallbackManager:[LPRequestCallbackManager sharedManager]];
```

Comment:
You do not need to rename the init here. Please keep the same code. All you need is this here.

sharedManager = [[self alloc] init];
callbackManager = [LPRequestCallbackManager sharedManager];